tools/commands/build_example.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+import re
+import subprocess
+import sys
+import os
+import commands.command_utils as command_utils
+
+
+def parse_args(args):
+    parser = command_utils.get_options_parser(
+        plugins=True, exclude=True, run_on_changed_packages=True, base_sha=True, command='build')
+    return parser.parse_args(args)
+
+
+def _build_examples(plugin):
+    name = plugin[plugin.rfind('/')+1:]
```

Comment:
There is a convenient api:

```suggestion
    name = os.path.basename(plugin)
```